Owncloud iOs Client.xcodeproj/project.pbxproj

```diff
@@ -630,6 +630,36 @@
 		905C594F1C50DA490051D223 /* ManageTouchID.m in Sources */ = {isa = PBXBuildFile; fileRef = 906177E11C4FABF600BBE648 /* ManageTouchID.m */; };
 		905C59501C50DA680051D223 /* ManageTouchID.m in Sources */ = {isa = PBXBuildFile; fileRef = 906177E11C4FABF600BBE648 /* ManageTouchID.m */; };
 		906177E21C4FABF600BBE648 /* ManageTouchID.m in Sources */ = {isa = PBXBuildFile; fileRef = 906177E11C4FABF600BBE648 /* ManageTouchID.m */; };
+		906414311D004719000AEFE5 /* available_offline_TB@2x.png in Resources */ = {isa = PBXBuildFile; fileRef = 9064142B1D004719000AEFE5 /* available_offline_TB@2x.png */; };
+		906414321D004719000AEFE5 /* available_offline_TB@3x.png in Resources */ = {isa = PBXBuildFile; fileRef = 9064142C1D004719000AEFE5 /* available_offline_TB@3x.png */; };
+		906414331D004719000AEFE5 /* available_offline_TB-filled.png in Resources */ = {isa = PBXBuildFile; fileRef = 9064142D1D004719000AEFE5 /* available_offline_TB-filled.png */; };
+		906414341D004719000AEFE5 /* available_offline_TB-filled@2x.png in Resources */ = {isa = PBXBuildFile; fileRef = 9064142E1D004719000AEFE5 /* available_offline_TB-filled@2x.png */; };
+		906414351D004719000AEFE5 /* available_offline_TB-filled@3x.png in Resources */ = {isa = PBXBuildFile; fileRef = 9064142F1D004719000AEFE5 /* available_offline_TB-filled@3x.png */; };
+		906414381D004BA1000AEFE5 /* available_offline_TB.png in Resources */ = {isa = PBXBuildFile; fileRef = 906414371D004BA1000AEFE5 /* available_offline_TB.png */; };
+		9064143F1D005B5C000AEFE5 /* file_new_server_version_available_icon.png in Resources */ = {isa = PBXBuildFile; fileRef = 906414391D005B5C000AEFE5 /* file_new_server_version_available_icon.png */; };
```

Comment:
Better update all icons and replace icons, for example this will be the old state off FileUpdatedIcon.
This will match with images in older versions